src/core_plugins/kibana/public/dashboard/actions/panels.js

```diff
@@ -0,0 +1,17 @@
+import { createAction } from 'redux-actions';
+
+export const deletePanel = createAction('DELETE_PANEL', panelId => panelId);
```

Comment:
This is not required, the default function in `createAction` is `_.indentity`, so you can simply write this (and other actions like this that simply pass there parameters to the reducer) like so:

```
export const deletePanel = createAction('DELETE_PANEL')
```

The only time you need to provide the function is if you need to map multiple arguments into a single object, or otherwise do something with the action's arguments before passing the values to the reducer (as you do below).